### PR TITLE
DS-443 DropdownMenu.Root changes

### PR DESCRIPTION
**Pull request by david-cho-lerat-sonarsource**

### CODE REVIEW

**Review by jeremy-davis-sonarsource — APPROVED** [review]
:muscle:

**Review by gregaubert — APPROVED** [review]

**Review by gregaubert — APPROVED** [review]
👍🏻

**Review by jeremy-davis-sonarsource — APPROVED** [review]
:+1:

File: src/components/dropdown-menu/DropdownMenu.tsx [hunk 1]
```diff
@@ -59,12 +61,15 @@ const DropdownMenuRoot = forwardRef<HTMLButtonElement, DropdownMenuRootProps>(
...
       items,
+      onClose,
+      onOpen,
       ...radixProps
     }: Readonly<DropdownMenuRootProps>,
     ref,
   ) => {
-    if (isDisabled || !isDefined(items)) {
+    if (isDisabled) {
       return <span>{children}</span>;
```
[the remark this answers is not in the document]
Comment (david-cho-lerat-sonarsource) [reply src/components/dropdown-menu/DropdownMenu.tsx:73]: true, an oversight on my part

**Pull request merged**